apps/financial-aid/web-osk/src/components/FileUploadResult/FileUploadResult.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from 'react'
+import {
+  ContentContainer,
+  StatusLayout,
+  Footer,
+} from '@island.is/financial-aid-web/osk/src/components'
+import { FileList } from '../../../../web-shared/src/components'
+import { FormContext } from '@island.is/financial-aid-web/osksrc/components/FormProvider/FormProvider'
+import { Box, Text } from '@island.is/island-ui/core'
+import * as styles from './fileUpload.treat'
+import { Colors } from '@island.is/island-ui/theme'
+
+interface Props {
+  subtitle: string
```

Comment:
Maybe camelcasing here subTitle?

Reply:
Subtitle is a single 🙈  see: https://dictionary.cambridge.org/dictionary/english/subtitle